packages/dds/merge-tree/src/snapshotV1.ts

```diff
@@ -333,6 +342,18 @@ export class SnapshotV1 {
 		// If the last segment in the walk was coalescable, push it now.
 		pushSeg(prev);
 
+		// if there are more than 500 segments removed by snapshot-time zamboni,
```

Comment:
i think the specifics of this comment aren't super important (they will quickly become out of date, and also we probably don't want to write approx numbers here in OSS as they come from more closed-source ecosystem).

I'd prefer to write something along the lines of expecting the log is potentially noisy, so we sample it to avoid high COGS.